README.md

```diff
@@ -1,24 +1,73 @@
-# README
+## Hotel Reservstion 🏩
+This application is a web application for final capstone project that you can find your favorite Hotel around the world and reserve it for a specific date, find information about a hotel at details page and cancel a reservation.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Project Documentation 📄
+- [Live Demo]()
+- [API Documentation](http://127.0.0.1:5000/api-docs/index.html)
 
-Things you may want to cover:
+## Team members: 4 (Dozie Udeagha, SaharSaba Amiri, Mean Mohammad, Bright Kweku Ahiadeke )
 
-* Ruby version
+# Built With 🛠️
+- Programming Languages: HTML, CSS, JavaScript, Ruby.
+- Frameworks: React, Redux, Rails and Bootstrap.
+- GitHub Linters.
+- Code Editor: VS Code.
 
-* System dependencies
+## Kanban Board
+- Click this [link](https://github.com/users/maenkhraisha/projects/6) to view the Kanaban Board
+- Click this [link](https://docs.google.com/document/d/1GcBpwqr2NsF9umMA4qthxuBxnEjcLagS8p_tt5CwHPc/edit?usp=sharing) to view the initial state of the kanban board
+- Click [here](https://github.com/maenkhraisha/hotel-reservation) for the main branch of the repo
+# Getting Started
+To get a local copy up and running follow these simple steps.
 
-* Configuration
+### Prerequisites
+- A web browser like Google Chrome.
+- A code editor like Visual Studio Code with Git and nodejs.
+- You can check if Git is installed by running the following command in the terminal.
+- `$ git --version`
+- Clone the repo from here.
+- `$ git clone https://github.com/maenkhraisha/hotel-reservation.git`
 
-* Database creation
+# Usage
+## Run locally
+- Make your way to the correct directory by running this command:
+- `$ cd hotel-reservation`
+- Run npm install
+- Then you can run this command to configure the app in your PC:
+- `$ rails db:drop db:create db:migrate db:seed`
+- For starting the server to see the app in windows you can run this command:
+- `foreman start`
```

Comment:
- [ ] Please, include ```bundle install``` so your users can get the ruby dependencies too.
- [ ] Using your instructions, when I try to run the app with ```foreman start```, I get into this:

![image](https://user-images.githubusercontent.com/57421823/201308662-468bf330-e663-4e8d-b564-4daac73578b4.png)

![image](https://user-images.githubusercontent.com/57421823/201308717-b988bc6b-cc5a-46b4-b4a1-dbc42b799cc5.png)

If the app is running fine for you, please add instructions and be more specific about how to run your app.  If this is a bug, kindly fix it so your app works as intended.

Also, webpack compiles with 2 errors:

![image](https://user-images.githubusercontent.com/57421823/201309516-1ca7ad64-a800-4a40-9462-9d1d1b9cf9ee.png)


![image](https://user-images.githubusercontent.com/57421823/201309434-83fd520e-60cc-439a-ad89-4e413e925216.png)



_note: This means I wasn't able to review the app functionality.  This will be done in the next review._

That way, your users will be able to run your app smoothly 💪